contracts/src/main/kotlin/com/r3/corda/lib/tokens/contracts/states/NonFungibleToken.kt

```diff
@@ -32,25 +32,27 @@ import net.corda.core.schemas.QueryableState
  *
  * @property token the [IssuedTokenType] which this [NonFungibleToken] is in respect of.
  * @property holder the [AbstractParty] which holds the [IssuedTokenType].
- * @param T the [TokenType].
+ * @property linearId the [UniqueIdentifier] which will uniquely identify this Token.
+ * @property tokenTypeJarHash the [SecureHash] which will pin the jar that provides the [TokenType].
```

Comment:
Maybe mention that devs should use default and there is no need to provide it

Reply:
OK